README.md

```diff
@@ -2,10 +2,10 @@ exists-sync
 ===========
 [![Build Status](https://travis-ci.org/ember-cli/exists-sync.svg)](https://travis-ci.org/ember-cli/exists-sync) [![Build status](https://ci.appveyor.com/api/projects/status/c05xyb4s80pn66yo?svg=true)](https://ci.appveyor.com/project/embercli/exists-sync)
 
-Replacement for [fs.existsSync()](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback), which is being deprecated: 
+Replacement for [fs.existsSync()](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback) (which has been un-deprecated in Node.js since [6.8.0](https://nodejs.org/en/blog/release/v6.8.0/)): 
```

Comment:
I would link to [PR](https://github.com/nodejs/node/pull/8364)

Reply:
Thanks @twokul! Updated! Makes sense - PR has much more information.